pkg/constants/constants.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package constants
+
+const (
+	LoadBalancerGCAnnotation = "capi-runtime-extensions.d2iq-labs.com/loadbalancer-gc"
```

Comment:
Let's use same domain as in API (https://github.com/d2iq-labs/capi-runtime-extensions/blob/main/api/v1alpha1/groupversion_info.go#L16):

```suggestion
	LoadBalancerGCAnnotation = "labs.d2iq.io/loadbalancer-gc"
```